source/codegen/templates/service.cpp.mako

```diff
@@ -129,6 +129,9 @@ ${mako_helper.define_simple_method_body(function_name=function_name, function_da
     catch (nidevice_grpc::LibraryLoadException& ex) {
       return ::grpc::Status(::grpc::NOT_FOUND, ex.what());
     }
+    catch (nidevice_grpc::ValueOutOfRangeException& ex) {
```

Comment:
Can this be conditional? Maybe on

```python
service_helpers.requires_checked_conversion(function)
```

which would currently be implemented as something like:
```python
return bool([p for p in function["parameters"] if is_input_array_that_needs_coercion(p)])
```